newdle/client/src/selectors.js

```diff
@@ -31,3 +31,22 @@ export const getFullTimeslots = state =>
     ...Object.entries(state.timeslots).map(([date, slots]) => slots.map(slot => `${date}T${slot}`))
   );
 export const getCreatedNewdle = state => state.createdNewdle;
+export const getNewdleTimeslots = state => (state.newdle && state.newdle.timeslots) || [];
+export const getNewdleDuration = state => state.newdle && state.newdle.duration;
+export const getNewdleParticipants = state => (state.newdle && state.newdle.participants) || [];
+export const getNumberOfParticipants = createSelector(
+  getNewdleParticipants,
+  participants => participants.length
+);
+export const getParticipantAvailability = createSelector(
+  getNewdleTimeslots,
+  getNewdleParticipants,
+  (timeslots, participants) => {
+    return timeslots.map(timeslot => {
+      const available = participants.filter(
+        participant => participant.answers[timeslot] === 'available'
```

Comment:
shouldn't `ifneedbe` be considered available as well? (at least I wouldn't consider those unavailable.. maybe yellow in the circular progress thing if it supports more than 2 states?)